apps/ledger-live-mobile/src/components/DeviceAction/InstallSetOfApps/index.tsx

```diff
@@ -63,10 +65,13 @@ const InstallSetOfApps = ({
   } = status;
 
   const onWrappedError = useCallback(() => {
-    if (onError && error) {
-      onError(error);
+    if (error) {
+      remountMe();
```

Comment:
Maybe add a comment to explain that it's to force `action.useHook` to re-run and reset the `status` value ;)

Reply:
Done